internal/provider/expressvpn/openvpnconf.go

```diff
@@ -14,7 +14,7 @@ func (p *Provider) OpenVPNConfig(connection models.Connection,
 		RemoteCertTLS: true,
 		AuthUserPass:  true,
 		Ciphers: []string{
-			openvpn.AES256cbc,
+			openvpn.AES128gcm, openvpn.AES256gcm, openvpn.AES256cbc,
```

Comment:
Can you flip these around to use aes256gcm then aes256cbc then aes128cbc, since aes128cbc is the less secured variant?